### DISCUSSION TITLE
Game integration

### DISCUSSION THREAD

**Discussion by RSchwoerer** [aquasarus/Starfield-Interactive-Smart-Slate#16] [General]
Just throwing this here as a start to game integration discussion.

There is a mod currently available that exposes the console as a HTTP endpoint. [Console API and Web Application](https://www.nexusmods.com/starfield/mods/4280)
Source: https://github.com/dranger003/sfse_plugin_console_api

Using this could be a great initial entry point to getting data from the game while playing if there is information that makes sense to read.

**Comment by aquasarus** [comment]
Definitely! SFSE is currently the definitive mod for extracting console output. I intend to use this to at least track the player's current location, but so far I have not found any other console commands that give us related survey information. This would also be an extra layer of dependency, so I'm half-hoping to hear more about Creation Kit's arrival date before I start working on this :P. Once that's here, there will be no need for such integrations.
